src/cli.rs

```diff
@@ -20,9 +22,15 @@ use strum::IntoEnumIterator;
 const COLOR_RESOLUTIONS: [&str; 5] = ["16", "32", "64", "128", "256"];
 pub const NO_BOTS_DEFAULT_REGEX_PATTERN: &str = r"(?:-|\s)[Bb]ot$|\[[Bb]ot\]";
 
+const STYLES: Styles = Styles::styled()
+	.header(AnsiColor::Yellow.on_default().effects(Effects::BOLD))
+	.usage(AnsiColor::Green.on_default().effects(Effects::BOLD))
+	.literal(AnsiColor::Green.on_default().effects(Effects::BOLD))
```

Comment:
```suggestion
	.header(AnsiColor::Yellow.on_default())
	.usage(AnsiColor::Green.on_default())
	.literal(AnsiColor::Green.on_default())
```
to match https://docs.rs/clap/latest/clap/builder/struct.Styles.html#example